src/server/pps/server/api_server.go

```diff
@@ -2344,10 +2369,17 @@ func getJobOptions(kubeClient *kube.Client, jobInfo *persist.JobInfo, jobShimIma
 		volumes:            volumes,
 		volumeMounts:       volumeMounts,
 		imagePullSecrets:   imagePullSecrets,
+		cpuFootprint:       jobInfo.Resources.GetCpu(),
+		memFootprint:       jobInfo.Resources.GetMemory(),
 	}, nil
 }
 
 func podSpec(options *jobOptions, jobID string, restartPolicy api.RestartPolicy) api.PodSpec {
+	mem, err := resource.ParseQuantity(options.memFootprint)
+	if err != nil {
+		mem = resource.MustParse(defaultMem)
+	}
+	cpu := resource.MustParse(fmt.Sprintf("%f", options.cpuFootprint))
```

Comment:
It seems like we are handling errors / defaults differently here for mem/cpu - is there a reason for that?